src/System.Net.Mail/tests/Functional/SmtpClientTest.cs

```diff
@@ -332,6 +332,82 @@ public async Task TestMailDeliveryAsync()
             }
         }
 
+        [Fact]
+        public void TestSmtpDeliveryFormatSevenBit() {
```

Comment:
I think this test is significantly more complicated than is necessary. Is it not possible to do something more like the test above this one?

Reply:
Eight pairs of parameter combinations, `DeliveryFormat` and `SMTPUTF8`, were tested. I want to compare the impact of `SmtpClient` modifications on existing functionality. It's a little redundant.

But now it's changed to test only one pair.